client/src/main/scala/drt/client/components/TerminalContentComponent.scala

```diff
@@ -126,17 +134,23 @@ object TerminalContentComponent {
                 )
               ),
               <.div(^.className := "exports",
+                if (state.activeTab == "arrivals") {
+                  <.div(^.className := s"filter-arrival",
+                    <.input.checkbox(^.checked := state.filterPassengerFlights, ^.onChange ==> filterPassengerFlightsToggle, ^.id := "toggle-arrival"),
+                    <.label(^.`for` := "toggle-arrival", "Passenger flights only"," ", passengerFlightsTooltip),
```

Comment:
should we name this "toggle-filter-non-passenger-flights" rather than "toggle-arrival"